src/Metadata/GetTables/DefaultGetTablesSerializer.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Keboola\DbExtractor\Metadata\GetTables;
+
+use Keboola\DbExtractor\Metadata\ValueObject\Column;
+use Keboola\DbExtractor\Metadata\ValueObject\Table;
+use Keboola\DbExtractor\Metadata\ValueObject\TableCollection;
+
+class DefaultGetTablesSerializer implements GetTablesSerializer
```

Comment:
V povodnom kode sa pre metadata ako vysledok `getTables` sync akcie ... a aj pre metadata, ktore idu do `manifestu` pouzivalo to iste pole ... bol tam z toho chaos, co je kde vlastne potrebne ... po konzultacii s JKotek sa ukazalo, ze vacsina parametrov posielanych cez `getTables` sa posiela zbytocne: https://keboola.slack.com/archives/CMTBFNLF8/p1588684823012800

Preto som vytvoril tento `GetTablesSerializer` aby to bolo jednoznacne a oddelene.